app/assets/javascripts/backbone/views/loan_questions_view.coffee

```diff
@@ -149,3 +155,20 @@ class MS.Views.LoanQuestionsView extends Backbone.View
         "<div class='loan-type-required'>#{loan_type}</div>"
       .join(' ') +
       "</div>"
+
+  permittedActionsHTML: (node) ->
+    if node.can_edit == true
+      $('.links-block').html()
+    else
+      $('.actions-disabled-block').html()
+
+  prepTooltips: ->
+    @$('.ms-tooltip').each (index, tip) ->
+      message = I18n.t('loan_questions.not_editable')
+
+      $(tip).addClass('ms-popover').popover
```

Comment:
I think if you change the inner `->` to `=>` above, you should be able to use `@$` here, which is preferable to `$`.

In order for `@` (which is the same as `this`) to refer to the right thing, you need the `=>`. Also happy to discuss this further if you like.